features/admin.applications.v1/pages/application-edit.tsx

```diff
@@ -506,6 +558,354 @@ const ApplicationEditPage: FunctionComponent<ApplicationEditPageInterface> = (
         return null;
     };
 
+    /**
+     * Resolves the application banner content.
+     *
+     * @returns Alert banner.
+     */
+    const resolveAlertBanner = (): ReactElement => {
+
+        const classes: any = classNames( { "application-outdated-alert-expanded-view": viewBannerDetails } );
```

Comment:
refrain from using any.

Reply:
I think we can replace `any` with `string` here.

Reply:
ce3f7f0210aea52dca4e1340286cc5ffcb701a43